tests/site_nav.js

```diff
@@ -92,33 +88,44 @@ describe('SiteNav', function() {
     });
   });
 
-  describe('handleChange()', function() {
-    it('should set the list aria hidden label to the opposite of checked',
-        function() {
-      var $toggle = this.$fixture.find('.js-toggle');
-      var $list = this.$fixture.find('#site-menu ul');
-      var testEv = {currentTarget: $toggle};
-
-      this.siteNav.handleChange(testEv);
-      expect($list.attr('aria-hidden')).to.not.equal($toggle.is(':checked'));
+  describe('toggle()', function() {
+    it('should show and hide the menu', function() {
+      var $nav = this.$fixture.find('.js-site-nav');
+      this.siteNav.toggle();
+      expect($nav.attr('class')).to.include('is-open');
+      this.siteNav.toggle();
+      expect($nav.attr('class')).to.not.include('is-open');
     });
   });
 
-  describe('hover()', function() {
-
-    beforeEach(function() {
-      this.$withDropdown = this.$fixture.find('.site-nav__item--with-dropdown');
-      this.$list = this.$fixture.find('#site-menu ul');
+  describe('toggleSublist()', function() {
+    it('should show and hide the sublist', function() {
+      var $sublistParent = this.$fixture.find('.js-sublist-parent');
+      var $sublistToggle = this.$fixture.find('.js-sublist-toggle');
+      var testEv = {target: $sublistToggle};
+      this.siteNav.toggleSublist(testEv);
+      expect($sublistParent.find('ul').attr('aria-hidden')).to.equal('false');
```

Comment:
Since `hideSublist` and `showSublist` each do a few different things, it might be worth adding helpers like `isOpen` and `isClosed` that check all the relevant DOM settings, plus the `isOpen` variable. See https://github.com/18F/fec-style/blob/master/tests/dropdowns.js#L15-L23 for examples.